specification/workloads/resource-manager/Microsoft.Workloads/preview/2021-12-01-preview/SAPVirtualInstance.json

```diff
@@ -286,6 +286,9 @@
           },
           "SAPVirtualInstances_Create_WithOSConfig_HA_AvZone": {
             "$ref": "./examples/sapvirtualinstances/SAPVirtualInstances_Create_WithOSConfig_HA_AvZone.json"
+          },
+          "SAPVirtualInstances_Create_Distributed_WithRestrictedDBVmSku": {
```

Comment:
let's make this descriptive, similar to 1st example in this list

Reply:
Modified to ""Creates a new SAP Virtual Instance for Distributed Non HA deployment type with restricted DB VM Sku"